docs/tdvt.md

```diff
@@ -176,132 +174,134 @@ This is a selection of the output:
+-|-|-|-|-
+Name | mydb | N/A | Your datasource name | Yes
+CommandLineOverride | -DLogLevel=Debug | N/A | Space separated list of arguments that are passed through unchanged to tabquerytool | No
+MaxThread | 6 | 6 | Controls maximum number of threads to a single datasource. Does not apply when running TDVT with multiple datsources | No
+TimeoutSeconds | 6000 | 3600 | Controls how long it takes for | No
+LogicalQueryFormat | bool_lower | N/A | The logical query we use | Yes
+
+### LogicalConfig
+This section is optional, and allows you to define your own logical query config if the existing ones do not work for your database. Some options are mutually exclusive.
+
+Option | Example Value | Default | Description | Required?
+-|-|-|-|-
+Name | my_logical_query | N/A | Name of your logical query | Yes
+tablename | SomePrefix_$dsName | N/A | The name of the table. $dsName will be substituted with Calcs or Staples. | No
+tablePrefix | [MySchema]. | N/A | Prefix that is appended to the table name | No
+tablePostfix | [MySchema]. | N/A | Postfix that is appended to the table name | No
+tablenameUpper | True | N/A | Set to true if the table name is all uppercase | No
+tablenameLower | True | N/A | Set to true if the table name is all lowercase | No
+bool_underscore | True | N/A | Set to true if the bool column name has underscores | No
+fieldnameDate_underscore | True | N/A | Set to true if the date column name has underscores | No
+fieldnameUpper | True | N/A | Set to true if the column names are all uppercase | No
+fieldnameLower | True | N/A | Set to true if the column names are all lowercase  | No
+fieldnameNoSpace | True | N/A | Set to true if the column names have no spaces | No
+fieldnameLower_underscore | True | N/A | Set to true if the column names have underscores and are all lowercase | No
+fieldnameUnderscoreNotSpace | True | N/A | Set to true if the column names replace spaces with underscores | No
+
+### [StandardTests]
+This is required to run the standard tests.
+
+This section also allows you to skip tests by excluding them. You can put in comma-separated string values that match part or all of a test name to exclude them. The asterisk works as a wildcard.
+
+Option | Example Value | Default | Description | Required?
+-|-|-|-|-
+LogicalExclusions_Calcs | string.right | N/A | Exclude logical tests that target the Calcs table | No
+LogicalExclusions_Staples | Filter.Trademark | N/A | Exclude logical tests that target the Staples table | No
+ExpressionExclusions_Standard | string.ascii,string.char | N/A | Exclude expression tests | No
+
+### [LODTests]
+Add this section to run LOD tests. This section is required.
+
+Option | Example Value | Default | Description | Required?
+-|-|-|-|-
+LogicalExclusions_Staples | Filter.Trademark | N/A | Exclude logical tests that target the Staples table | No
+ExpressionExclusions_Calcs | string.ascii,string.char | N/A | Exclude expression tests that target the Calcs table| No
```

Comment:
I didn't see any examples of `ExpressionExclusions_Calcs` usage today

Reply:
I copy/pasted the keyword from the example, guessing it's from a really old version of TDVT if it isn't being used in favor of ExpressionExclusions_Standard. I'll change this.